src/modules/utils/panel/screens/3dprinter/WatchScreen.cpp

```diff
@@ -249,7 +252,17 @@ void WatchScreen::display_menu_line(uint16_t line)
             }
             break;
         }
-        case 1: THEPANEL->lcd->printf("X%4d Y%4d Z%7.2f", (int)round(this->pos[0]), (int)round(this->pos[1]), this->pos[2]); break;
+        case 1:
+            if ( THEPANEL->is_playing() ) {
```

Comment:
I can do that.  Two questions:
1.  I was thinking to add the option as a sub-item under `panel` (such as `panel.display_extruder`), since if you have no panel, displaying the extruder value is irrelevant.  Would that be acceptable?
2.  Should I add that config as an example to any of the sample config files?